src/ripple/app/paths/impl/BookStep.cpp

```diff
@@ -240,7 +230,7 @@ class BookPaymentPolicy
     }
 
     Quality
-    qualityUpperBound(ReadView const& v,
+    qualUpperBound(ReadView const& v,
         Quality const& ofrQ,
         Book const& book,
         AccountID const& strandDst,
```

Comment:
`book`, `strandDst`, and `ownerPaysTransferFee` all all member variables and don't need to be passed in as parameters.